src/vs/workbench/browser/quickopen.ts

```diff
@@ -247,20 +248,38 @@ export class EditorQuickOpenEntry extends QuickOpenEntry implements IEditorQuick
 	}
 
 	public run(mode: Mode, context: IEntryRunContext): boolean {
-		if (mode === Mode.OPEN) {
+		const hideWidget = mode === Mode.OPEN;
+
+		let backgroundOpts;
+		if (mode === Mode.OPEN_IN_BACKGROUND) {
+			backgroundOpts = objects.assign({}, {
+				pinned: true,
+				preserveFocus: true
+			});
+		}
+
+		if (mode === Mode.OPEN || mode === Mode.OPEN_IN_BACKGROUND) {
 			let sideBySide = context.keymods.indexOf(KeyMod.CtrlCmd) >= 0;
+			let opts = this.getOptions();
+			if (backgroundOpts) {
+				opts = objects.assign(opts || {}, backgroundOpts);
```

Comment:
Any reason why we cannot merge the `getOptions` options with the `(<IResourceInput>input).options`?  I know you previously mentioned it was not done.

If we can, it might be a nice approach to put the options logic in `getOptions`

```ts
	public getOptions(mode: Mode): EditorOptions {
		let opts = {};
		if (mode === Mode.OPEN_IN_BACKGROUND) {
			opts = { pinned: true, preserveFocus: true };
		}

		return EditorOptions.create(opts);
	}
```

for now I'll just 

```ts
	public getOptions(): EditorOptions {
		return EditorOptions.create({});
	}
```